docs/docs/rest-api/public/api/v2/types/podStatus.raml

```diff
@@ -159,6 +163,10 @@ types:
       containers?:
         type: ContainerStatus[]
         description: status for each running container of this instance.
+      specReference?:
```

Comment:
Like the idea of pointing to the uri.
Do we have a runSpecVersion on the instance level?